example/hello.cc

```diff
@@ -38,6 +38,14 @@ void run() {
     exit(1);
   }
 
+  // Validate.
+  std::cout << "Validating module..." << std::endl;
+  bool valid = wasm::Module::validate(store, binary);
+  if (!valid) {
```

Comment:
```suggestion
  if (!wasm::Module::validate(store, binary)) {
```
(for symmetry with the C version)